packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/AdobeController.java

```diff
@@ -15,16 +16,19 @@
 public class AdobeController {
 
 	@Value("${adobe.api-key}")
-	String key;
+	String adobeKey;
```

Comment:
Can't `adobeKey` just be defined with the local value in some local application.properties, so that we don't need to have this sort of logic in the controller class? `application-local.properties`?

Reply:
The point is to be able to use the local `hmi-client` with the _staging_ or _dev_ `hmi-server`.
So adding it to the local properties will not solve the problem.